html/changelogs/geeves-mining_modernization.yml

```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - backend: "Fixed up a lot of mining-related things in the backend. You might notice some changes in-game."
+  - tweak: "Locked abandoned crates found in the mine now has 15 attempts to hack instead of 10."
```

Comment:
```suggestion
  - tweak: "Locked abandoned crates found in the mine now has 15 attempts to hack instead of 10."
  - tweak: "Destroying artifacts while mining now longer has long range effects on many people."
```